content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -130,119 +92,102 @@ spec:
+The CNI DaemonSet will detect and evict pod stuck at such state. The new pod starts up should have traffic redirection set up properly.
+This mitigation is enabled by default and can be turned off by setting `values.cni.repair.enabled` to false.
 
 ### Traffic redirection parameters
 
-To redirect traffic in the application pod's network namespace to/from the Istio proxy sidecar, the Istio
-CNI plugin configures the namespace's iptables.  The following table describes the parameters to the
-redirect functionality.  To override the default values for the parameters, set the corresponding
-application pod annotation key.
-
-| Annotation Key | Values | Default | Description |
-|----------------|--------|---------|-------------|
-| `sidecar.istio.io/inject` | `true`, `false` | `true` | Indicates whether the Istio proxy sidecar should be injected. If present and `false`, the Istio CNI plugin doesn't configure the namespace's iptables for the pod. |
-| `sidecar.istio.io/status` | | | Annotation created by Istio's sidecar injection. If missing, the Istio CNI plugin doesn't configure the pod namespace's iptables. |
-| `sidecar.istio.io/interceptionMode` | `REDIRECT`, `TPROXY` | `REDIRECT` | The iptables redirect mode to use. |
-| `traffic.sidecar.istio.io/includeOutboundIPRanges` | `<IPCidr1>,<IPCidr2>,...` | `"*"` | Comma separated list of IP ranges in CIDR form to redirect to the sidecar proxy.  The default value of `"*"` redirects all traffic. |
-| `traffic.sidecar.istio.io/excludeOutboundIPRanges` | `<IPCidr1>,<IPCidr2>,...` | | Comma separated list of IP ranges in CIDR form to be excluded from redirection.  Only applies when `includeOutboundIPRanges` is `"*"`. |
-| `traffic.sidecar.istio.io/includeInboundPorts` | `<port1>,<port2>,...` | Pod's list of `containerPorts` | Comma separated list of inbound ports for which traffic is to be redirected to the Istio proxy sidecar.  The value of `"*"` redirects all ports. |
-| `traffic.sidecar.istio.io/excludeInboundPorts` | `<port1>,<port2>,...` | | Comma separated list of inbound ports to be excluded from redirection to the Istio sidecar proxy.  Only valid when `includeInboundPorts` is `"*"`. |
-| `traffic.sidecar.istio.io/excludeOutboundPorts` | `<port1>,<port2>,...` | | Comma separated list of outbound ports to be excluded from redirection to Envoy. |
-| `traffic.sidecar.istio.io/kubevirtInterfaces` | `<ethX>,<ethY>,...` | | Comma separated list of virtual interfaces whose inbound traffic (from VM) will be treated as outbound. |
-
-### Logging
-
-The Istio CNI plugin runs in the container runtime process space.
-Due to this, the `kubelet` process writes the plugin's log entries into its log.
+To redirect traffic in the application pod's network namespace to/from the Istio proxy sidecar,
+the Istio CNI plugin configures the namespace's iptables.
+You can adjust traffic redirection parameters using pod annotations, such as ports and IP ranges to be included or excluded from redirection.
```

Comment:
nit:
```suggestion
You can adjust traffic redirection parameters using the same pod annotations as normal, such as ports and IP ranges to be included or excluded from redirection.
```

Maybe a better way to phrase it, but just saying that its not CNI-specific. Just that CNI also supports them like normal